ivy/array/general.py

```diff
@@ -817,32 +862,24 @@ def default(
         """
         return ivy.default(self, default_val, catch_exceptions, rev, with_callable)
 
-    def stable_pow(
-        self: ivy.Array,
-        exponent: Union[Number, ivy.Array, ivy.NativeArray],
-        min_base: float = None,
-    ) -> Union[ivy.Array, ivy.NativeArray]:
+    def supports_inplace(self: ivy.Array) -> bool:
```

Comment:
Please revert this change as it's not related to the function you're reformatting